doc/governance/proposals.rst

```diff
@@ -1,35 +1,41 @@
 Proposing and Voting for a Proposal
 ===================================
 
-.. warning::
+.. note::
     See :doc:`/governance/js_gov` for pointers on converting from Lua to JS.
 
-This page explains how members can submit and vote for proposals.
+Summary
+-------
 
 Proposals are submitted as JSON documents, which if resolved successfully are applied atomically to the KV state.
 
-Ballots are submitted as JavaScript modules, executed transactionally, and are able to read from the current KV state but not write to it.
-Each vote script is given the proposal as a list of actions, and returns a Boolean indicating whether it supports or rejects it.
+Ballots are submitted as JavaScript modules exporting a single `vote()` function, executed transactionally, and are able to read from the current KV state but not write to it.
+Each vote script is given the proposal as a list of actions, and returns a Boolean value indicating whether it supports or rejects it.
```

Comment:
"as a list of actions" is not correct. We say proposals are JSON documents, so perhaps "given the proposal body" or "the raw proposal body" or "the JSON document from the proposal".

Reply:
I think I want to talk about both, because while we offer flexibility here, we in practice push in a single direction in all our samples.

Reply:
Sure, but this suggests `vote()` takes a list. It doesn't, it takes an `ArrayBuffer`, and even after `JSON.parse` that is an object not a list. So if we're not trying to describe the signature here, this can be vague but correct like "Each vote script is given the proposal, and returns...".